04/app.js

```diff
@@ -4,6 +4,26 @@ const user = {
     born: {
         day: '14',
         month: '04',
-        year: '1985'
+        year: '1985',
     }
-}
\ No newline at end of file
+}
+
+const now = new Date();
+
+const check = checkDate(user, now)
+
+if (check === true) {
+    console.log('ma urodziny :-)');
+} else {
+    console.log('nie ma urodzin ;-(');
+}
+
+
+function checkDate(user, date) {
+    if (user.born['year'] == date.getFullYear() || user.born['month'] == date.getMonth() || user.born['day'] == date.getDate()) {
```

Comment:
Ten warunek nie jest poprawny, ponieważ:
1) od kiedy rok jest istotny czy ktoś ma urodziny?
2) chyba warunki trzeba spełniać łącznie czyli && (AND, logiczne i)
3) sprawdź czy zwracane dni oraz miesiące są zgodne z porównaniem np. `'04'`,